app/models/shelter.rb

```diff
@@ -1,5 +1,9 @@
 class Shelter < ApplicationRecord
-  validates_presence_of :name, :address, :city, :state, :zip
-  has_many :pets
-  has_many :shelter_reviews 
+  validates_presence_of :name,
+                        :address,
+                        :city,
+                        :state,
+                        :zip
+  has_many :pets, dependent: :destroy
+  has_many :shelter_reviews, dependent: :destroy
```

Comment:
`dependent: :destroy` allows us to delete a shelter by destroying all reviews and pets associated with it. This also completes User Story 28. Will have to revisit (probably add a conditional) when working on User Story 26.

Reply:
Thanks for catching this! I had this on my to-do list after my project review since Mike pointed it out.